### DISCUSSION TITLE
DylibExample build failed, can't compile to dylib

### DISCUSSION THREAD

**Discussion by newNcy** [wasmerio/wasmer#4533] [Q&A]
![image](https://github.com/wasmerio/wasmer/assets/19602632/fd497a9c-c7ce-4672-8920-8430beec5654)

**Comment by linear** [comment]
<p><a href="https://linear.app/wasmer/issue/RUN-136/dylibexample-build-failed-cant-compile-to-dylib">RUN-136 DylibExample build failed, can't compile to dylib</a></p>

**Comment by syrusakbary** [comment]
We deprecated the use of multiple engines as part of the Wasmer 3.0 release to simplify the process: https://github.com/wasmerio/wasmer/blob/master/CHANGELOG.md#300-alpha4---20220728

It's still possible to do something similar in the future, though.
Can you expand on your use case so we can see how to solve it best @newNcy ? Thanks!
